src/components/NcColorPicker/NcColorPicker.vue

```diff
@@ -159,32 +154,34 @@ export default {
 </docs>
 
 <template>
-	<NcPopover v-bind="$attrs" v-on="$listeners" @apply-hide="handleClose">
+	<NcPopover @apply-hide="handleClose">
 		<template #trigger>
 			<slot />
 		</template>
 		<div class="color-picker"
 			:class="{ 'color-picker--advanced-fields': advanced && advancedFields }">
-			<transition name="slide" mode="out-in">
-				<div v-if="!advanced" class="color-picker__simple">
-					<button v-for="(color, index) in palette"
-						:key="index"
-						:style="{'background-color': color }"
-						class="color-picker__simple-color-circle"
-						:class="{ 'color-picker__simple-color-circle--active' : color === currentColor }"
-						type="button"
-						@click="pickColor(color)">
-						<Check v-if="color === currentColor"
-							:size="20" />
-					</button>
+			<Transition name="slide" mode="out-in">
+				<div>
+					<div v-if="!advanced" class="color-picker__simple">
+						<button v-for="(color, index) in palette"
+							:key="index"
+							:style="{'background-color': color }"
+							class="color-picker__simple-color-circle"
+							:class="{ 'color-picker__simple-color-circle--active' : color === currentColor }"
+							type="button"
+							@click="pickColor(color)">
+							<Check v-if="color === currentColor"
+								:size="20" />
+						</button>
+					</div>
+					<Chrome v-if="advanced"
+						v-model="currentColor"
+						class="color-picker__advanced"
+						:disable-alpha="true"
+						:disable-fields="!advancedFields"
+						@update:modelValue="pickColor" />
 				</div>
-				<Chrome v-if="advanced"
-					v-model="currentColor"
-					class="color-picker__advanced"
-					:disable-alpha="true"
-					:disable-fields="!advancedFields"
-					@input="pickColor" />
-			</transition>
+			</Transition>
```

Comment:
This additional `<div>` wrapper breaks the transition

Reply:
Indeed. I added it because I got a `VueCompilerError: <Transition> expects exactly one child element or component.`, but that was triggered by using `v-if="!advanced"` and `v-if="advanced"` instead of `v-if/v-else`. I fixed it now and the transition works again.